src/Molecules/Export/Export.jsx

```diff
@@ -73,52 +71,11 @@ const Export = (
             'data-testid': 'export-filename',
           }}
           value={filename}
-          onChange={event => setFilename(event.target.value)}
+          onChange={event => onChange({ filename: event.target.value, format })}
         />
       </FormControl>
-      {
-        formats.length > 0 &&
-          <FormControl>
-            <InputLabel>
-              <Trans transKey="global.export.format"/>
-            </InputLabel>
-            <Select
-              inputProps={{
-                'data-testid': 'export-format',
-              }}
-              value={format}
-              onChange={event => setFormat(event.target.value)}
-            >
-              <option value="" disabled>
-                {t('global.chooseOption')}
-              </option>
-              {formats.map(({ value, label }) => <option value={value} key={value}>{label}</option>)}
-            </Select>
-          </FormControl>
-      }
-      {
-        extraOptions.length > 0 &&
-          <FormControl component="fieldset">
-            <FormLabel component="legend">
-              <Trans transKey="global.export.dataToExportLabel"/>
-            </FormLabel>
-            <RadioGroup
-              aria-label="exportScope"
-              name="exportScope"
-              value={exportScope}
-              onChange={event => setExportScope(event.target.value)}
-            >
-              {extraOptions.map(({ label, value }) =>
-                <FormControlLabel
-                  key={value}
-                  value={value}
-                  control={<Radio inputProps={{ 'data-testid': `export-extra-option-${value}` }}/>}
-                  label={label}
-                />
-              )}
-            </RadioGroup>
-          </FormControl>
-      }
+      <Formats formats={formats} value={format} onChange={newFormat => onChange({ filename, format: newFormat })} />

```

Comment:
No need for a new name, it's already contextualized by the arrow function
```suggestion
      <Formats formats={formats} value={format} onChange={format => onChange({ filename, format })} />
```